src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx

```diff
@@ -18,39 +18,6 @@ freshnessValidatedDate: never
 
 New Relic's Kubernetes plugin for log forwarding simplifies sending logs from your cluster to New Relic logs. It uses a standalone Docker image and runs as a DaemonSet, seamlessly collecting logs for centralized analysis and troubleshooting. Forwarding your Kubernetes logs to New Relic will give you enhanced <InlinePopover type="logs" /> capabilities to collect, process, explore, query, and alert on your log data.
 
-## Enable Kubernetes for log management [#enable-process]
-
-To forward your Kubernetes logs to New Relic with our plugin:
-
-1. [Install the New Relic Kubernetes integration](/install/kubernetes). This integration includes the Kubernetes plugin for logs.
-
-2. Optionally, you can further tune your installation in [the guided install](/install/kubernetes) using the numerous configuration options available in the [`newrelic-logging` repository](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging#configuration). However, we recommend the standard setup, as it's valid for most users.
-
-    <img
-        title="Guided install from the UI"
-        alt="Guided install from the UI"
-        src="/images/kubernetes_screenshot-crop_gather-log-data.webp"
-    />
-
-    <figcaption>
-        Go to <DNT>**[one.newrelic.com](https://one.newrelic.com/) > Integrations & Agents**</DNT> and click the <DNT>**Guided install**</DNT> tile. Then select <DNT>**Kubernetes**</DNT> and <DNT>**Guided**</DNT>.
-    </figcaption>
-
-    <Callout variant="important">
-
-    If you're [using a Kubernetes secret](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-logging/values.yaml#L8-L25) to store the New Relic <InlinePopover type="licenseKey" />, the `newrelic-logging` chart defaults to sending logs to the US API endpoint. If the license key belongs to an EU or FedRAMP account, and a secret is used for key storage, you must update the endpoint setting with the appropriate value from the [API reference docs](/docs/logs/log-api/introduction-log-api/#endpoint). Here's an example of how to set this for EU accounts:
-
-    ```yml
-        newrelic-logging:
-            enabled: true
-            endpoint: https://log-api.eu.newrelic.com/log/v1
-    ```
-    </Callout>
-
-3. Generate some traffic and wait a few minutes, then [check your account](#find-data) for data.
-
-<InstallFeedback />
-
 ## Fluent Bit output plugin [#fluent-bit]
 
 New Relic has a [Fluent Bit](https://fluentbit.io/) [output plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to forward your logs to New Relic log management. This plugin is also provided in a standalone Docker image that can be installed in a Kubernetes cluster in the form of a DaemonSet, also known as the Kubernetes plugin.
```

Comment:
@nbaenam I think the general link to Fluent Bit is distracting since readers probably already know about Fluent Bit. You might drop that link. 

The second link is to the GitHub repository for the Fluent Bit plugin. As a reader, how should I decide which links to click on? We offer both a GitHub repository link and another docs link in the logs section that seems to do the same thing: https://deploy-preview-18479--docs-website-netlify.netlify.app/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding/#fluentbit-plugin). I suppose they all eventually work, but it seems to present a mixed message. Why not just offer one or the other and not give readers an unnecessary choice?

Reply:
@nbaenam I know this is beyond the scope of this PR, but at some point, could you clarify when I'd want to use the Fluent Bit plugin? For example, there's a sub-section called "Additional metric details" that looks to me like you'd get log forwarding automatically via Fluent Bit is you installed `nri-bundle` with the Helm chart. Is that true? In other words, would I need to install the Fluent Bit plugin if I've already installed `nri-bundle` using the Helm chart?

Reply:
Hey @rhetoric101, I posted the question to the help-logging channel.

Reply:
Hi @rhetoric101, this ticket (NR-310861) has been opened to review the content related to the Fluent Bit plugin. Do you mind approving it for now? I'll open a new PR when I have their feedback.